.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.4
```

Comment:
I'm thinking it might not be something we want committed? It appears to have been added in a commit that was just merging with master: c0811eee68216a5251fed8bac73a45472244af05

When I pull the branch, I get the message:
```
error: The following untracked working tree files would be overwritten by merge:
	.ruby-version
Please move or remove them before you merge.
```